Readme.md

```
@@ -186,16 +237,17 @@ For DynamicComponent-based hosts, inherit `ComponentWizardViewModel<TResult>` an
 - **Composability** - Mix and match reusable steps
 - **Testability** - Business logic isolated from Blazor components
 
-### Class Hierarchy
+### Core Types
 ```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)